src/DynamoCoreWpf/Views/Core/GeometryScalingPopup.xaml.cs

```diff
@@ -0,0 +1,113 @@
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+using Dynamo.ViewModels;
+using ViewModels.Core;
+using System.Linq;
+using Dynamo.Logging;
+using System;
+using Res = Dynamo.Wpf.Properties.Resources;
+using System.Windows.Input;
+
+namespace Dynamo.Views
+{
+    public partial class GeometryScalingPopup : Popup
+    {
+        private GeometryScalingViewModel viewModel;
+        private DynamoViewModel dynamoViewModel;
+        
+        public GeometryScalingPopup(DynamoViewModel dynViewModel)
+        {
+            InitializeComponent();
+
+            var defaultWorkspace = dynViewModel.Workspaces.FirstOrDefault();
+
+            if (defaultWorkspace != null && viewModel == null && defaultWorkspace.GeoScalingViewModel != null)
+                viewModel = defaultWorkspace.GeoScalingViewModel;
+            DataContext = viewModel;
+            dynamoViewModel = dynViewModel;
+        }
+
+        /// <summary>
+        /// This event is generated every time the user clicks a Radio Button in the Geometry Scaling section
+        /// The method just get the Radio Button clicked and saves the ScaleValue selected
+        /// This are the values used for the scales:
+        /// - 2 - Small
```

Comment:
Well I think this is still `1 - Small` right?

Reply:
Well, this comment is trying to give information to developer related to which value could contain the scaleFactor variable, if you see to the ConvertUIToScaleFactor() method is using a formula so that if we pass the 0 index from UI (the first button - small) then the formula returns -2 and that's the value stored in ScaleValue.
![image](https://user-images.githubusercontent.com/61755417/214113685-3496e16b-4136-4051-a90d-698c98efc29b.png)
This is a table of the values, I hope this make it more clear.
![image](https://user-images.githubusercontent.com/61755417/214114467-bd778b1d-c1b8-4d3a-afe4-f8ab2716a8f1.png)